sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2825,7 +2818,7 @@
           "type": "string"
         },
         "type": {
-          "description": "Required.\r\nThe type of job to execute.",
+          "description": "Required. The type of job to execute.",
```

Comment:
Remove the "Required" statement, since the field will be marked as required later